testsuite/features/secondary/min_centos_monitoring.feature

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@centos_minion
+Feature: Monitor SUMA environment with Prometheus on a CentOS minion
+  In order to monitore SUSE Manager server
+  As an authorized user
+  I want to enable Prometheus exporters
+
+  Scenario: Pre-requisite: enable Prometheus exporters repository on the CentOS minion
+    And I enable repository "tools_pool_repo" on this "ceos_ssh_minion"
+
+  Scenario: Apply Prometheus exporter formulas on the CentOS minion
+    Given I am on the Systems overview page of this "ceos_ssh_minion"
+    When I follow "Formulas" in the content area
+    And I should see a "Choose formulas:" text
+    And I should see a "Monitoring" text
+    And I check the "prometheus-exporters" formula
+    And I click on "Save"
+    Then I should see a "Formula saved" text
+
+  Scenario: Configure Prometheus exporter formula on the CentOS minion
+    Given I am on the Systems overview page of this "ceos_ssh_minion"
+    When I follow "Formulas" in the content area
+    And I follow "Prometheus Exporters" in the content area
+    And I should see a "Enable and configure Prometheus exporters for managed systems." text
+    And I check "node_exporter#enabled"
+    And I check "apache_exporter#enabled"
+    And I check "postgres_exporter#enabled"
```

Comment:
Ideally, we don't use internals like HTML or javascript internals. We use some clear English name, and convert it to the internal inside the step definition.

Look at we do for Retail formulas to have an example.